nautobot/ipam/choices.py

```diff
@@ -102,6 +102,23 @@ class IPAddressTypeChoices(ChoiceSet):
     )
 
 
+#
+# VRFs
+#
+
+
+class VRFStatusChoices(ChoiceSet):
```

Comment:
Should this be added to `nautobot.extras.management.STATUS_CHOICESET_MAP` like the other statuses?

Do we need a data migration (similar to `0055_softwareimage_softwareversion_data_migration.py`) to call `populate_status_choices()` so that these statuses get auto-created on upgrade?